dashboards-observability/public/components/event_analytics/explorer/visualizations/config_panel/config_panes/config_controls/config_thresholds.tsx

```diff
@@ -35,6 +36,15 @@ export const ConfigThresholds = ({
   props,
 }: any) => {
   const addButtonText = '+ Add threshold';
+  const AddButtonTextWrapper = () =>
+    props?.maxLimit && !isEmpty(vizState) && vizState.length === props.maxLimit ? (
+      <EuiToolTip position="top" content="Only one threshold can be applied">
+        <>{addButtonText}</>
```

Comment:
Do you really need this additional fragment?

Reply:
Actually tooltip accepts JSX elements only, that's why we have to wrap it using Fragment.